packages/base-types/src/node/utils/event.ts

```diff
@@ -19,6 +22,12 @@ export interface IntentEvent extends BaseEvent, SlotMappings {
   intent: string;
 }
 
+export interface AlexaEvent extends BaseEvent {
+  type: EventType.ALEXA;
+  name: string;
+  mappings: Array<{ var: Nullable<string>; path: string }>;
```

Comment:
`var` is a keyword in JS, is there perhaps a different name we could use that won't conflict with that & potentially cause issues with tools? if not, it might just be fine to keep it as-is

Reply:
makes sense, i'm fine with keeping it like this then 🙂